### PR TITLE
[o-mr0] sony: common: init: Move common stuff from platform

### PR DESCRIPTION
**Pull request by humberos**
commonize init stuff

Signed-off-by: Humberto Borba <humberos@gmail.com>
Change-Id: I902d56149e0546fa52789a537bcc85ca30ee5190

### CODE REVIEW

**Comment by humberos** [comment]
oh.... there are more common lines
please hold on, I will update this PR.

File: rootdir/init.common.rc [hunk 1]
```diff
@@ -161,6 +177,55 @@ on early-boot
     chown system system /sys/class/graphics/fb0/msm_fb_persist_mode
 
 on boot
+    # WLAN and BT MAC
+    chown system system /sys/devices/soc/soc:bcmdhd_wlan/macaddr
```
Comment (bartcubbins) [inline rootdir/init.common.rc]: @humberos I don't think that this is common for Yoshino platform too

Comment (humberos) [reply rootdir/init.common.rc]: I think it will not affect boot process.
We will get only warnings 
anyway thanks for the hint

**Comment by humberos** [comment]
@jerpelea 
as you can see I've added bootanim back but at on post-fs-data which is fine on tone.

File: rootdir/init.common.rc (hunk 1 above)
Comment (jerpelea) [reply rootdir/init.common.rc]: please leave this on loire and tone (bcm platforms)

Comment (humberos) [reply rootdir/init.common.rc]: Done!

**Comment by humberos** [comment]
oh
[    3.880181] init: decode_uid: Unable to find UID for 'persist_rw'; returning UINT_MAX
[    3.880280] init: decode_uid: Unable to find UID for 'persist_rw'; returning UINT_MAX

Shall I change it to system (as AOSP) ?

**Comment by humberos** [comment]
Also fixed

[    2.716776] init: init.common.srv.rc: 58: user requires 1 argument
[    2.716937] init: init.common.srv.rc: 102: user requires 1 argument
[    2.717088] init: init.common.srv.rc: 140: user requires 1 argument

**Comment by jerpelea** [comment]
camera-daemon is deprecated and removed

**Comment by humberos** [comment]
Done

**Pull request merged**